azure-pipelines.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   steps:
   - task: GoTool@0
     inputs:
-      version: '1.14'
+      version: '1.14.7'
```

Comment:
Does that azure task support `1.14.x`? That's what we use for github actions. Strange if not...

Reply:
I didn't find that in the docs.
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/go-tool?view=azure-devops
But might work, I will try

Reply:
Or if it supports semver patterns... `^1.14.7` :man_shrugging:

Reply:
1.14.x worked